webapp/src/components/DonationsDashboard/DonationsDashboard.js

```diff
@@ -311,81 +385,259 @@ const DonationsDashboard = ({ isDesktop }) => {
+            {role === "donor" &&
+              <Box>
+                <Typography className={classes.sectionTitle}>
+                  {t('donations.sendTokens')}
+                </Typography>
+                <Divider className={classes.dividerSection} />
+                <Typography className={classes.sectionText}>
+                  {t('donations.toSendTo')}
+                </Typography>
+              </Box>
+            }
+            {role === "lifebank" &&
+              <Typography className={classes.sectionText}>
+                {t('donations.toSendToDonor')}
+              </Typography>
+            }
+            <Box className={classes.boxTexfield}>
+              <form autoComplete="off">
+                <TextField
+                  autoFocus
+                  className={classes.inputText}
+                  id="filled-basic"
+                  label={t('donations.enterSponsorUsername')}
+                  variant="filled"
+                  value={payload.to || ''}
+                  onChange={(event) =>
+                    handleSetField('to', event.target.value)
+                  }
+                />
+                <QrReaderModal />
+              </form>
+            </Box>
+            {errorMessage && (
+              <Alert
+                className={classes.alert}
+                severity="error"
+                action={
+                  <IconButton
+                    aria-label="close"
+                    color="inherit"
+                    size="small"
+                    onClick={() => setErrorMessage(null)}
+                  >
+                    <CloseIcon fontSize="inherit" />
+                  </IconButton>
+                }
+              >
+                {errorMessage}
+              </Alert>
+            )}
+            {success && transferResult && (
+              <Alert
+                className={classes.alert}
+                severity="success"
+                action={
+                  <IconButton
+                    aria-label="close"
+                    color="inherit"
+                    size="small"
+                    onClick={() => setSuccess(false)}
+                  >
+                    <CloseIcon fontSize="inherit" />
+                  </IconButton>
+                }
+              >
+                {t('donations.succesful')}
+              </Alert>
+            )}
+            <Box className={classes.boxButtonSendToken}>
+              <Button className={classes.sendTokenButton} variant="contained" color="secondary"
+                onClick={hanndlerTransferTokens}
+                disabled={
+                  !payload.to ||
+                  !payload.quantity ||
+                  !payload.memo ||
+                  loading
+                }
+              >
+                {t('donations.sendToken')}
+              </Button>
+            </Box>
+          </Box>
+        }
+      </>
+    )
+  }
+
+  const QrReaderModal = () => {
+    return (
```

Comment:
Unnecessary return statement here. Remember that when a JSX element just has one return in its body we can omit it and do lile const JSXElement = () => (
    JSX here ...
)